packages/core/helper-plugin/src/utils/isAbsoluteUrl/index.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Check if an Url provided is Absolute or not
+ * @param {string|undefined} url
+ * @returns {boolean} the response to the question if the provided url is absolute or not
+ */
+const isAbsoluteUrl = (url) => {
+  const absoluteUrlRegex = new RegExp('^(?:[a-z+]+:)?//', 'i');
```

Comment:
I would argue an URL starting with a slash is also absolute.

Reply:
ok I will change the name